packages/editor/src/css/style_mosaico_tools.less

```diff
@@ -836,6 +868,14 @@
     padding: 1.2em;
   }
 
+  /* Style for the search empty state */
+  #personalized-blocks-list .empty-state-search {
+    text-align: center;
+    font-size: 1.2em;
+    color: gray;
+    padding: 1.2em;
```

Comment:
not em for spacing (unless you have good reasons ^^), prefer to use rem

Reply:
i took the same padding of the others text messages classes, I fixed all of them now